src/test/java/nl/tudelft/ewi/devhub/webtests/views/NotificationView.java

```diff
@@ -0,0 +1,28 @@
+package nl.tudelft.ewi.devhub.webtests.views;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Arjan on 14-6-2017.
+ */
+public class NotificationView extends AuthenticatedView {
+
+    private static final String NOTIFICATION1 = "/html/body/div/table/tbody/tr[1]/td";
+    private static final String NOTIFICATION2 = "/html/body/div/table/tbody/tr[2]/td";
+    private static final String NET_ID = "admin1";
+
+    public NotificationView(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    public void invariant() {
+        assertTrue("Wrong Path", currentPathEquals("/notifications/" + NET_ID));
+        assertNotNull(By.xpath(NOTIFICATION1));
```

Comment:
This is kind of dependent on the configuration on the bootstrapper, which in theory could be configurable per test. Please verify instead that the notification table is there, and expose a mechanism to list potential notifications (which are then asserted in the tests itself).